operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-link
```

There is no indication what namespace this goes into - I assume it goes into the deployment namespace (e.g. "istio-system") - we need to add that to the metadata.

We also need more metadata here to be consistent with the rest of the resources created by the operator.

```suggestion
  name: kiali-app-link
  namespace: {{ kiali_vars.deployment.namespace }}
  labels:
    app: kiali
    version: {{ kiali_vars.deployment.version_label }}
```